custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc_daily.py

```diff
@@ -78,11 +90,11 @@ def aggregation_query(self):
             ('cases_person_adolescent_girls_15_18_all',),
             ('daily_attendance_open', '0'),
             ('num_awcs',),
-            ('num_launched_states', '0'),
-            ('num_launched_districts', '0'),
-            ('num_launched_blocks', '0'),
-            ('num_launched_supervisors', '0'),
-            ('num_launched_awcs', '0'),
+            use_agg_awc_data_or_zero('num_launched_states'),
```

Comment:
to keep this in line with our convention in other files like agg_awc, i think this should be
`('num_launched_states', use_agg_awc_data_or_zero('num_launched_states'))`
where the function determines the second value of the tuple but the first is still a clear string

Reply:
that way second element of the tuple will always have something right? and we will have to change the logic below `[col[1] if len(col) > 1 else col[0] for col in columns]),`. Or probably if  `use_agg_awc_data_or_zero` just returns same column name as a string making the final tuple as `('num_launched_states', 'num_launched_states')` which should work But, That also does not look like going with convention. Do you see that fine?
M I missing something?

Reply:
yeah, thats what we do in agg_awc, just return the string if necessary. so that we dont have to change the logic below.